lib/gitspindle/github.py

```diff
@@ -1170,29 +1170,30 @@ def pull_request(self, opts):
             print("Pull request %d created %s" % (pull.number, pull.html_url))
             return
 
-        body = self.find_template(repo, 'PULL_REQUEST_TEMPLATE')
+        pr_template = self.find_template(repo, 'PULL_REQUEST_TEMPLATE')
```

Comment:
`find_template()` already does `try_decode()`, however, it's not turning our bytes into an actual (Unicode) string.